src/screens/search_results/SearchResultsHeader.tsx

```diff
@@ -47,42 +47,53 @@ const SearchResultsHeader: React.FC<SearchResultsHeaderProps> = ({
     query,
     viewState,
     setViewState,
-    showDetails,
     setShowDetails,
     setHash,
     disableControls = false,
 }) => {
     const [sortState, setSortState] = useState<'alpha' | 'rev' | 'none'>('none');
+    const [filterState, setFilterState] = useState<FilterProps>({ showType: 'none' });
     const windowSize = useWindowSize();
+    const unsortedShows = localStorage.getItem('streamabilityUnsortedResults');
 
-    const handleViewToggle = (view: 'grid' | 'list') => {
-        setViewState?.(view);
-        localStorage.setItem('streamabilityView', view);
-        setHash?.(Math.random());
-    };
-
-    const handleSortAlpha = () => {
-        const sortedShows = sortShowsAlphaAsc(showDetails || []);
-        setShowDetails?.(sortedShows);
-        setSortState('alpha');
-        setHash?.(Math.random());
-    };
+    useEffect(() => {
+        if (!unsortedShows) return; // should never be false
```

Comment:
We could add back in the `LOG.error` here just so dev's could know if this happens.